tags/tags.toml

```
@@ -803,3 +803,18 @@ Version 13 is still in development, the branch moves quickly.
 • [Preview update guide](<https://deploy-preview-680--discordjs-guide.netlify.app/additional-info/changes-in-v13.html>) (use `CTRL` + `F` to search for the old method or property)
 *\\🚧 This is still a draft and may not be up to date at all times*
 """
+
+[master-events]
+keywords = ["master-events", "master events", "me", "v13-events", "v13 events"]
+content = """
+On master branch, `message` and `interaction` events have been renamed to `messageCreate` and `interactionCreate` respectively, to bring the library in line with Discord's naming conventions.
+
+Don't worry - the old names still work, but you'll receive a Deprecation Warning until you switch over.
+
+```diff
+- <Client>.on('message', message => { ... });
++ <Client>.on('messageCreate', message => { ... });
+- <Client>.on('interaction', interaction => { ... });
++ <Client>.on('interactionCreate', interaction => { ... });
+```
```

```suggestion
The `message` and `interaction` events have been renamed to `messageCreate` and `interactionCreate` respectively, to bring the library in line with Discord's naming conventions.

The old events still work, but you'll receive a deprecation warning until you switch over.

```diff
- client.on('message', message => { ... });
+ client.on('messageCreate', message => { ... });
- client.on('interaction', interaction => { ... });
+ client.on('interactionCreate', interaction => { ... });
```
```

---

Wouldn't this fall under using the guide interaction to point users to the v13 changes guide?